vars.tf

```diff
@@ -170,3 +176,9 @@ variable "asset_types" {
     "storage.googleapis.com/Bucket",
   ]
 }
+
+variable "security_findings_filter" {
```

Comment:
Can you add a comment explaining that the default is here because the API requires a filter?